src/MigrateCommand.php

```diff
@@ -334,6 +335,80 @@ private function updatePackages(array $packages) : void
         }
     }
 
+    private function updateConfigPipeline() : void
+    {
+        $this->output->write('<info>Updating config pipeline...</info>');
+
+        $files = glob('config/autoload/*.php');
```

Comment:
I think just to be safe we can check all files here. Everywhere we use `config/autoload/middleware-pipeline.global.php` but it's possible that file has been renamed.

Reply:
If that file is renamed, it might have been moved as well (e.g. config/pipeline.php). If files have been renamed or moved the migration might fail anyway. However I don't see any harm in scanning with glob.